charts/stable/homarr/templates/_secrets.tpl

```diff
@@ -0,0 +1,10 @@
+{{- define “homarr.secrets” -}}
```

Comment:
Ugh fix quotes please I’m on mobile lol

Reply:
Oh I see what I did I copied your modification onto the whole file lol